docs/userguide/targets.md

```diff
@@ -1,18 +1,87 @@
 
-BayBE currently supports a single type of target, called [`NumericalTarget`](baybe.targets.numerical.NumericalTarget), specifically designed for numerical modeling.
+Below is a visualization of possible choices for `transformation`, where `lower` and
+`upper` are the entries provided via `bounds`:
+![Transforms](../_static/target_transforms.svg)
 
-## Numerical targets
+### MIN and MAX mode
+Here are two examples for simple maximization and minimization targets:
+```python
+from baybe.targets import NumericalTarget, TargetMode, TargetTransformation
 
-Besides their name, numerical targets have the following attributes.
-* **The optimization mode:** This mode dictates whether we want to minimize or maximize the target or whether we want to match a specific value.
-* **Bounds:** Define optional bounds that constrain the range of target values.
-* **A bound transformtaion function:** When bounds are provided, a bound transformation function is used to map target values onto the [0,1] interval. This is only applicable in minimization and maximization mode.
+max_target = NumericalTarget(
+    name="Target_1",
+    mode=TargetMode.MAX,  # can also be provided as string "MAX"
+)
 
-Targets are used on all of the examples, mostly in either minimization or maximization mode. For an example on how the matching mode can be effectively utilized, we refer to the dedicated example provided [here](./../../examples/Multi_Target/desirability).
\ No newline at end of file
+min_target = NumericalTarget(
+    name="Target_2",
+    mode="MIN",  # can also be provided as TargetMode.MIN
+    bounds=(0, 100),  # optional
+    transformation=TargetTransformation.LINEAR,  # optional, will be applied if bounds are not None
+)
+```
+
+### MATCH mode
+If you want to match a desired value, the `TargetMode.MATCH` mode is the right choice.
+In this mode, `bounds` are required and different transformations compared to `MIN`
+and `MAX` modes are allowed.
+
+Assume we want to instruct BayBE to match a value of 50 in a target.
+We simply need to choose the bounds so that the midpoint is the desired value.
+The spread of the bounds interval defines how fast the acceptability of a measurement
+falls off away from the match value, also depending on the choice of `transformation`.
+
+In the example below, `match_targetA` will treat all values `< 45` and `> 55` as
+equally bad, while `match_targetB` is more forgiving in that it chooses a bell curve
+transformation instead of triangular, and also uses a wider interval of bounds.
+Both targets are configured to match the midpoint of `bounds` as optimal, in this case
+a value of `50`:
+
+```python
+from baybe.targets import NumericalTarget, TargetMode, TargetTransformation
+
+match_targetA = NumericalTarget(
+    name="Target_3A",
+    mode=TargetMode.MATCH,
+    bounds=(45, 55),  # mandatory in MATCH mode
+    transformation=TargetTransformation.TRIANGULAR,  # optional, applied if bounds are not None
```

Comment:
not sure I agree with that, passing `transformation` as argument (which is what this `optional` refers to) is always optional, even for match targets. Whats not optional for match targets are the bounds.

This comment does not aim to explain the inner workings and that there is always (ie not optionally) a transformation applied

Reply:
I understand and agree with your point, but mine was a slightly different one: should we make "transformation" mandatory for "MATCH" mode? Because the currently it can only be internally omitted for "MAX/MIN". If you choose "MATCH", you need to provide bounds, but if you provide bounds, you need to also provide a transform. Hence, "MATCH" implies using a transform, and if the user doesn't specify it, a default one will be used. And that's something that the user might not even be aware of.

Reply:
> Hence, "MATCH" implies using a transform, and if the user doesn't specify it, a default one will be used> 

that is literally how all other parameters with default values work, why do you feel the need to change/mention this for this particular case?

Reply:
Yeah, probably you are right and I'm overthinking :D Ok, let's close it!